### ISSUE TITLE
be able to turn off greeting

### ISSUE THREAD

**Issue by h4ck3rm1k3** [gehaxelt/python-rss2irc#36]
feature, turn off greeting

**Comment by h4ck3rm1k3** [comment]
Here is my hack :+1: 

```
index c7c52c3..283457d 100644
--- a/bot.py
+++ b/bot.py
@@ -41,7 +41,7 @@ class IRCBot(irc.bot.SingleServerIRCBot):

     def on_join(self, connection, event):
         """Say hello to other people in the channel. """
-        connection.privmsg(self.__config.CHANNEL, "Hi, I'm " + Colours('3',str(connection.get_nickname())).get() + " yo\
ur bot. Send " + Colours(self.num_col,"!help").get() +" to get a list of commands.")
+        #connection.privmsg(self.__config.CHANNEL, "Hi, I'm " + Colours('3',str(connection.get_nickname())).get() + " y\
our bot. Send " + Colours(self.num_col,"!help").get() +" to get a list of commands.")
         self.__on_connect_cb()

```

**Comment by gehaxelt** [comment]
Hi @h4ck3rm1k3 
does this correlate with #29 ? 
If so, I implemented a bugfix in #32 which will send the welcome message once to every new joining user via a private message. 

Maybe we don't need a separate switch for this?

**Comment by h4ck3rm1k3** [comment]
yes
